src/templates.ts

```diff
@@ -1672,37 +1673,58 @@ class TeamPane {
     this.statsEl.appendChild(totalBaseStatEl);
 
     const awakeningsToDisplay = [
-      Awakening.SKILL_BOOST,
-      Awakening.TIME,
-      Awakening.SOLOBOOST,
-      Awakening.BONUS_ATTACK,
-      Awakening.BONUS_ATTACK_SUPER,
-      Awakening.SBR,
-      Awakening.RESIST_POISON,
-      Awakening.RESIST_BLIND,
-      Awakening.RESIST_JAMMER,
-      Awakening.RESIST_CLOUD,
-      Awakening.RESIST_TAPE,
-      Awakening.OE_FIRE,
-      Awakening.OE_WATER,
-      Awakening.OE_WOOD,
-      Awakening.OE_LIGHT,
-      Awakening.OE_DARK,
-      Awakening.OE_HEART,
+      [
+        Awakening.SKILL_BOOST,
+        Awakening.TIME,
+        Awakening.SOLOBOOST,
+        Awakening.BONUS_ATTACK,
+        Awakening.BONUS_ATTACK_SUPER,
+        Awakening.L_GUARD
+      ],
+      [
+        Awakening.SBR,
+        Awakening.RESIST_POISON,
+        Awakening.RESIST_BLIND,
+        Awakening.RESIST_JAMMER,
+        Awakening.RESIST_CLOUD,
+        Awakening.RESIST_TAPE
+      ],
+      [
+        Awakening.OE_FIRE,
+        Awakening.OE_WATER,
+        Awakening.OE_WOOD,
+        Awakening.OE_LIGHT,
+        Awakening.OE_DARK,
+        Awakening.OE_HEART
+      ],
+      [
+        Awakening.ROW_FIRE,
+        Awakening.ROW_WATER,
+        Awakening.ROW_WOOD,
+        Awakening.ROW_LIGHT,
+        Awakening.ROW_DARK,
+        Awakening.RECOVER_BIND
+      ]
     ];
-    for (const awakening of awakeningsToDisplay) {
-      const container = create('span') as HTMLSpanElement;
-      const awakeningIcon = create('span', ClassNames.AWAKENING) as HTMLSpanElement;
-      const [x, y] = getAwakeningOffsets(awakening);
-      awakeningIcon.style.backgroundPosition = `${AwakeningEditor.SCALE * x}px ${AwakeningEditor.SCALE * y}px`;
-
-      container.appendChild(awakeningIcon);
-      const aggregatedAwakeningCount = create('span') as HTMLSpanElement;
-      aggregatedAwakeningCount.innerText = 'x0';
-      this.aggregatedAwakeningCounts.set(awakening, aggregatedAwakeningCount);
-      container.appendChild(aggregatedAwakeningCount);
-      this.statsEl.appendChild(container);
+    const awakeningTable = create('table', ClassNames.AWAKENING_TABLE) as HTMLTableElement;
+    for (const awakeningSet of awakeningsToDisplay) {
+      const awakeningRow = create('tr') as HTMLTableRowElement;
+      for (const awakening of awakeningSet) {
+        const container = create('td') as HTMLSpanElement;
```

Comment:
You're making an HTMLTableCellElement with td.  Although It doesn't vastly change the outcome, please fix.